app/editor/menus/image.tsx

```diff
@@ -72,6 +74,15 @@ export default function imageMenuItems(
     {
       name: "separator",
     },
+    {
+      name: "commentOnImage",
```

Comment:
This reflects `commentOn${nodeType}` format. Can follow the same pattern for enabling comments on other node types.